scripts/bert/utils.py

```diff
@@ -63,18 +66,22 @@ def swap(token, target_idx, token_to_idx, idx_to_token, swap_idx):
     assert None not in idx_to_token
     assert len(set(idx_to_token)) == num_tokens
 
-    vocab_dict = {}
-    vocab_dict['idx_to_token'] = idx_to_token
-    vocab_dict['token_to_idx'] = token_to_idx
-    vocab_dict['reserved_tokens'] = reserved_tokens
-    vocab_dict['unknown_token'] = unknown_token
-    vocab_dict['padding_token'] = padding_token
-    vocab_dict['bos_token'] = None
-    vocab_dict['eos_token'] = None
-    json_str = json.dumps(vocab_dict)
-    converted_vocab = gluonnlp.Vocab.from_json(json_str)
+    bert_vocab_dict = {}
```

Comment:
Also need to update the sha-1 for hosted vocab file for pretrained models

Reply:
In progress.